SPECS/python-ptyprocess/python-ptyprocess.spec

```diff
@@ -0,0 +1,141 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%global srcname ptyprocess
+Summary:        Run a subprocess in a pseudo terminal
+Name:           python-%{srcname}
+Version:        0.7.0
+Release:        12%{?dist}
+License:        ISC
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/pexpect/ptyprocess
+Source0:        https://pypi.io/packages/source/p/%{srcname}/%{srcname}-%{version}.tar.gz
+BuildArch:      noarch
+%if %{with_check}
+BuildRequires:  python3-pytest
+%endif
+
+%description
+Launch a subprocess in a pseudo terminal (pty), and interact with both the
+process and its pty.
+
+%package -n python3-ptyprocess
+Summary:        Run a subprocess in a pseudo terminal
+BuildRequires:  python3-devel
+
+%description -n python3-ptyprocess
+Launch a subprocess in a pseudo terminal (pty), and interact with both the
+process and its pty.
+
+%prep
+%autosetup -n %{srcname}-%{version}
+
+%build
+python3 setup.py build
+
+%install
+python3 setup.py install --skip-build --prefix=%{_prefix} --root=%{buildroot}
+
+%check
+python3 -m pytest -v
+
+%files -n python3-ptyprocess
+%license LICENSE
+%doc README.rst
+%{python3_sitelib}/ptyprocess/
+%{python3_sitelib}/ptyprocess-*.egg-info
+
+%changelog
+* Mon May 10 2021 Thomas Crain <thcrain@microsoft.com> - 0.7.0-1
```

Comment:
It looks like we've also updated the version - it would be good to mention that in the changelog as well along with why we did it.